SU2_CFD/src/output_su2.cpp

```diff
@@ -312,3 +312,77 @@ void COutput::WriteCoordinates_Binary(CConfig *config, CGeometry *geometry, unsi
   
 }
 
+void COutput::WriteProjectedSensitivity(CConfig *config,
+                                        CGeometry *geometry,
+                                        unsigned short val_iZone,
+                                        unsigned short val_nZone) {
+  
+  unsigned short iVar;
+  unsigned long iPoint, iElem, iNode;
+  unsigned long *LocalIndex = NULL;
+  bool *SurfacePoint = NULL;
+  string filename, fieldname;
+  
+  filename = config->GetDV_Sens_Filename();
```

Comment:
Why would want to overwrite the input file? We usually have distinct file names for input and for output (as e.g. for meshes and solutions). Maybe it's more generic if we have it like this here, too?

Reply:
Agreed. I am going to submit one or two small final changes to this PR, and I will update this.

Reply:
Nice :)